### DISCUSSION TITLE
Avenues Khabar [NP]

### DISCUSSION THREAD

**Discussion by Carlinhos027** [iptv-org/community#365] [Show and tell]
### Channel ID

AvenuesKhabar.np

### Stream URL

http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/channel/UCMDzPE_7fcZSRJgpwIVor_A/live

### Notes

(Not 24/7)

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/channel/UCMDzPE_7fcZSRJgpwIVor_A/live (offline)
